KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -51,14 +50,14 @@ final class KarhooBookingViewController: UIViewController, BookingView {
         super.viewDidLoad()
         forceLightMode()
         setupMapView(reverseGeolocate: journeyInfo == nil)
+        mapView.set(userMarkerVisible: true)
```

Comment:
What's the reason for moving this from view did load to view will appear?

Reply:
There is no difference between one of the other steps, so I moved it, so it's updated only once.